docs/uma-tokenholders/guidence-on-collateral-currency-addition.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Guidance for the Addition of Collateral Currencies
+sidebar_label: Guidance for the Addition of Collateral Currencies
+---
+Collateral currencies can be added through the collateral currency UMIP template available on Github, by adding 
+
+ - the name of the token to the be added 
+ - the contract address of the token  together with a link to the address on etherscan
+ - the amount that should be added to the store as final fee expressed in units of the intended collateral currency.   
+_This should equate to approximately $400 at the time the UMIP is submitted.  If the token is not directly listed on Coingecko and/or Coinmarketcap, please add how the estimated dollar value of the token was obtained in the Rationale section._
+
+***Note**: rebasing tokens cannot be added as collateral currencies as they do not work with UMA's factory contracts*
+
+____________
+The below provides some guidelines when using collateral currencies in an UMA contract
+
+### Stablecoins
+
+Consider carefully whether to treat stablecoins as their intended peg, or as the current market value.
+
+### Non-traded tokens
+
+Some tokens, such as interest bearing tokens or liquidity tokens may have no direct market.  
+
+If using such tokens in a contract which requires collateral monitoring, such as Expiring Multiparty Contracts (EMP), please ensure that there is a clear way for liquidity to be obtained should it be required.  It may be useful to add this information to the UMIP, under the security section.
+
+### Volatility
+
+Where a proposed collateral type has high volatility, it is suggested that a higher store fee than the recommended $400 may be appropriate.
+
+Where this is the case, this should be noted in the rationale section together with a justification for the value of the store fee.
```

Comment:
Not saying this information is wrong at all, but doesn't this and the other recommendations for inclusion into the UMIP contradict the attempt to streamline the collateral currency UMIPs? 

If we are recommending to include this, there should be some sort of defined volatility metric when this is required.

Reply:
Idea was really to leave it open to add or not.

Reply:
I guess I just have a small issue with the wording. I think saying that it "should be noted" (but without saying exactly which situations require that it be noted) doesn't seem very clear to me. If I was writing an umip based off of this, I wouldn't really know when I should include this information or not. 

Also do we do this today? I don't think we propose higher final fees than $400 for volatile currencies. Do you think we should? I would probably say no, as volatility just as strongly implies that the final fee could be quickly overprices as opposed to underpriced.

Also nit: all references to a "store fee" should be changed to "final fee". This is it's canonical name, and is used in previous umips so I think it is less confusing.